src/user/settings.rs

```diff
@@ -1,27 +1,107 @@
+use std::collections::HashMap;
+use std::fs;
+use std::path::Path;
+
 use config::{Config, Environment, File};
 use serde::{Deserialize, Serialize};
 
-#[derive(Debug, Deserialize, Clone, Serialize)]
-pub struct Settings {
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct GlobalSettings {
```

Comment:
i wanted to call it global because it's gonna be used across projects whereas project is... per project. users have "accounts" and the accounts are definitely configured per project (see account_id). this is me sorting through how cloudflare does accounts.

would renaming it to GlobalUserSettings help? i think the global bit is still important

Reply:
Yes I didn't got that initially, GlobalUserSettings make sense to me, thanks.